### DISCUSSION TITLE
Migration Guide to Latest

### DISCUSSION THREAD

**Discussion by venkateshpullaganti** [manojadams/mui-forms#76] [Q&A] [answered]
Hi @manojadams 

We want to upgrade to the latest version of the MUI forms. Can you please provide the migration guide

In one system we are currently using:      `mui-forms: 1.0.27`
In another system, we are using: `"@manojadams/metaforms": "^3.0.23",`

**Comment by venkateshpullaganti** [comment]
We are planning to migrate as soon as possible @manojadams

**Comment by venkatesh1146** [comment]
Hi @manojadams 

We are planning to update to the latest version of MUI forms, but we don't have any migration guide or the schema changes that we need to do. 


Can you please provide us the migration guide ?

**Reply by manojadams** [reply, marked as answer]
Ok @venkatesh1146.
Documentation of latest version is in progress. You can track the progress below
- https://github.com/manojadams/mui-forms/issues/94

Migration guide will be provided from version 1 to 2 in the same.
